src/python/ddapp/ikplanner.py

```diff
@@ -1352,6 +1352,15 @@ def computePostureCost(self, pose):
         return np.sum(np.abs(pose - nominalPose)*cost)
 
 
+    def runMultiRRT(self, grHand, qStart, xGoal, objectGrasped = False):
```

Comment:
It cascades om way you have ikPlanner.runIkTraj() call ik.runIkTraj().
Again, I would like to merge runMultiRRT() into runIjTraj() and remove it from here, later if thats ok.